readme.md

```diff
@@ -207,25 +207,25 @@ $ touch app/views/artists/index.html.erb
 - Inside `app/views/artist/index.html.erb`:
   - Just put: `<h1>All Artists</h1>`
 
-Great, now let's refresh the page and there should be no more errors so we know everything has been wired up correctly!
+Great, now let's refresh the page. There shouldn't be any more errors so we know everything has been wired up correctly!
 
 However, let's think about what we really want to see when we visit this page. When we visit `/artists`, we expect to see information about all artists!
 
 So now the question is how can we dynamically generate a view for all of our data about artists?
 
-**(STWG)**: What does it mean to have our view update dynamically?
+What does it mean to have our view update dynamically?
 
 > Means we can write abstractive code that will update and render the appropriate data for all the records in our query. Can you imagine if you were a store with 5000 products, having to write out or hard code all those listings?
 
 Lets use [Facebook](http:/facebook.com) as an example:
 
-Facebook has like 4 billion users. They definitely do not have 4 billion profile HTML pages lying around -- that would be a ridiculous amount of data. Instead, they have ONE HTML page lying around, and when you look at someone's profile they insert the appropriate data into the appropriate places in that HTML, and send your computer the result.
+Facebook has about 2 billion active users. They definitely do not have 2 billion HTML profile pages lying around -- that would be a ridiculous amount of data. Instead, they have one HTML page, and when you look at someone's profile they insert the appropriate data into the appropriate places in that HTML, and send your computer the result.
 
-## Break (10 / 65)
+## Break (10 min)
 
-### Instance Variables (10 / 70)
+### Instance Variables (10 min)
 
-Reviewing what we learned in our intro to Rails class, controllers are responsible for fetching the correct data from our models. But now we need a way to pass this data to be displayed in our views.
+Reviewing what we learned in our Intro to Rails class, controllers are responsible for fetching the correct data from our models. But, now we need a way to pass this data to be displayed in our views.
 
 The way Rails solves this problem is by extending the functionality of Ruby's **instance variables**
 
```

Comment:
don't need to explicitly write this in the lesson, but can say this is like what we saw before with sinatra